vendor/comit_i/version

```diff
@@ -1 +1 @@
-v0.1.1-alpha
+v0.1.2-alpha
```

Comment:
```suggestion
v0.2.0-alpha
```

🙃 Not that it matters too much for a pre-release but I think it should be a minor version release rather than a patch. Maybe even a major version (first digit) because we did changes that are not backwards compatible.

Reply:
ok!